src/tribol/mesh/MfemData.cpp

```diff
@@ -891,6 +915,221 @@ std::unique_ptr<mfem::BlockOperator> MfemJacobianData::GetMfemBlockJacobian( con
   return block_J;
 }
 
+#ifdef TRIBOL_USE_ENZYME
+
+std::unique_ptr<mfem::BlockOperator> MfemJacobianData::GetMfemBlockJacobianEnzyme( const MethodData& method_data ) const
```

Comment:
Can you clarify here or elsewhere whether this routine gets every single jacobian contribution for the full tangent stiffness?

Reply:
Clarified naming and comment